### PR TITLE
Add config ipi_3az_rhcos_3m_3w_3i_vault_v2_external_with_namespace

### PR DESCRIPTION
**Pull request by petr-balogh**

### CODE REVIEW

**Comment by petr-balogh** [comment]
Verification job:
https://url.corp.redhat.com/64344f6

**Review by vavuthu — DISMISSED** [review]

File: conf/deployment/aws/ipi_3az_rhcos_3m_3w_3i_vault_v2_external_with_namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+DEPLOYMENT:
+  allow_lower_instance_requirements: false
```
Comment (vavuthu) [inline conf/deployment/aws/ipi_3az_rhcos_3m_3w_3i_vault_v2_external_with_namespace.yaml:3]: need to add   
   infra_nodes: True

**Review by vavuthu — APPROVED** [review]

**Comment by petr-balogh** [comment]
Re-triggered verification after config changes:
https://url.corp.redhat.com/9cfb403

**Comment by petr-balogh** [comment]
New verification after rebase on fixed master:
https://url.corp.redhat.com/b2228fb

**Review by vavuthu — APPROVED** [review]

**Review by dahorak — APPROVED** [review]

**Pull request merged**